docs.md

```diff
@@ -316,27 +385,25 @@ The rest are just local overrides for the same options found in the cbindgen.tom
   / etc(if any). The idea is for this to be used to annotate the operator with
   attributes, for example:
 
-```rust
-/// cbindgen:eq-attributes=MY_ATTRIBUTES
-#[repr(C)]
-pub struct Foo { .. }
-```
-
-Will generate something like:
+  ```rust
+  /// cbindgen:eq-attributes=MY_ATTRIBUTES
+  #[repr(C)]
+  pub struct Foo { .. }
+  ```
 
-```
-  MY_ATTRIBUTES bool operator==(const Foo& other) const {
-    ...
-  }
-```
+  Will generate something like:
 
-Combined with something like:
+  ```
+    MY_ATTRIBUTES bool operator==(const Foo& other) const {
+      ...
+    }
+  ```
 
-```
-#define MY_ATTRIBUTES [[nodiscard]]
-```
+  Combined with something like:
 
-for example.
+  ```
+  #define MY_ATTRIBUTES [[nodiscard]]
+  ```
```

Comment:
These changes seem unrelated? Seems ok, but maybe worth splitting into its own commit / PR? Or is it some sort of autoformatting i'm not aware of?

Reply:
I was just fixing broken indentation in the existing .md, so that bullets would render correctly. If you hide whitespace changes the diff disappears:
![image](https://github.com/user-attachments/assets/324b1256-2909-48e7-af22-bdcb56e5d4fe)

Happy to split it out if it's too annoying, tho.